### ISSUE TITLE
[Question] Targeting .NET 7.0 Directly

### ISSUE THREAD

**Issue by SuRGeoNix** [Ruslan-B/FFmpeg.AutoGen#254]
Hi @Ruslan-B, I was wondering if we could get better performance by compiling directly to .NET 6+ instead of just using .NET Standard.

I asked this in [stackoverflow](https://stackoverflow.com/questions/70487668/mixing-net-5-6-clarification) long time ago but I never got an answer.

**Comment by zgabi** [comment]
FFmpeg.AutoGen has no dependency, so without writing some .NET 6 specific code it makes no sense to add .NET6 tfw. The result will be the same anyway.

As far as I know when a package "A" targets netstandard, and it is using a package "B", which has netstandard and net6 version (and the net6 veresion is really contains some net6 improvements) it will use the netstandard version of "B". So in this case it makes sense to add net6 tfw to package "A", since it will inculde the net6 version of "B". But this is not the case with FFmpeg.AutoGen. 

Anyway I support adding net6+ target if there is a useful improvement in FFmpeg.AutoGen which needs net6+.

**Comment by SuRGeoNix** [comment]
@zgabi Yes, I'm talking about the case that an App or Package written in .NET6+ is targeting FFmpeg.Autogen. In this case (I think!) you will get much better performance if you use a .NET6+ FFmpeg.Autogen compiled dll.

**Comment by Ruslan-B** [comment]
.NET 7 has newer interop API - I am working now on adding .NET7 as another target platform.

**Comment by Ruslan-B** [comment]
Will be addressed with the upcoming NativeInterop package targeting .NET 6+.
